src/engine/guidance/post_processing.cpp

```diff
@@ -298,6 +303,10 @@ std::vector<RouteStep> postProcess(std::vector<RouteStep> steps)
 
 void trimShortSegments(std::vector<RouteStep> &steps, LegGeometry &geometry)
 {
+#if OSRM_POST_PROCESSING_PRINT_DEBUG
```

Comment:
Can we please not have commented out debug logging? I know this might be convenient for debugging but littering the code base with it makes it harder to read and maintain.